extensions/gdx-tools/src/com/badlogic/gdx/tools/etc1/ETC1AtlasCompressionResult.java

```diff
@@ -0,0 +1,59 @@
+

```

Comment:
It is difficult to build a meaningful dialog here since there is no specific person assigned to evaluate this PR.